From dd48ce5e8af867858c11f815a198c96e4b0d55ad Mon Sep 17 00:00:00 2001
From: Peter Colberg <peter.colberg@intel.com>
Date: Wed, 6 Dec 2023 16:51:45 -0500
Subject: [PATCH] configs: enable CONFIG_INTEL_IOMMU_SVM

This implicitly enables CONFIG_IOMMU_SVA which is a prerequisite of
CONFIG_FPGA_DFL_PCI_SVA. The option CONFIG_INTEL_IOMMU_SVM is enabled
in current RHEL 8 and Ubuntu 22.04 kernels.

Signed-off-by: Peter Colberg <peter.colberg@intel.com>
---
 configs/dfl-config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/dfl-config b/configs/dfl-config
index b5d7b1c4e249..a7a5e93fb918 100644
--- a/configs/dfl-config
+++ b/configs/dfl-config
@@ -8,6 +8,7 @@ CONFIG_FPGA_REGION=m
 
 CONFIG_FPGA_DFL=m
 CONFIG_FPGA_DFL_PCI=m
+CONFIG_INTEL_IOMMU_SVM=y
 CONFIG_FPGA_DFL_PCI_SVA=m
 CONFIG_FPGA_DFL_PLATFORM=m
 CONFIG_FPGA_DFL_AFU=m
